web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCertificatesAvailableFunctionsImpl.java

```diff
@@ -192,26 +192,7 @@ public GetCertificatesAvailableFunctionsImpl(AuthoritiesHelper authoritiesHelper
             )
         );
 
-        if (!certificate.getMetadata().getPatient().isProtectedPerson()) {
-            resourceLinks.add(
-                ResourceLinkDTO.create(
-                    ResourceLinkTypeDTO.PRINT_CERTIFICATE,
-                    PRINT_NAME,
-                    PRINT_DRAFT_DESCRIPTION,
-                    true
-                )
-            );
-        } else {
-            resourceLinks.add(
-                ResourceLinkDTO.create(
-                    ResourceLinkTypeDTO.PRINT_CERTIFICATE,
-                    PRINT_NAME,
-                    PRINT_DRAFT_DESCRIPTION,
-                    PRINT_PROTECTED_PERSON_BODY,
-                    true
-                )
-            );
-        }
+        addPrintResourceLink(certificate, resourceLinks);
```

Comment:
Consider having a getMethod that returns a print resourcelink instead of passing in the list and adding it inside the method.